newrelic-agent/src/test/java/com/newrelic/agent/AgentHelper.java

```diff
@@ -68,7 +68,7 @@
 import java.util.StringTokenizer;
 import java.util.logging.Level;
 
-import static org.mockito.Matchers.anyString;
+import static org.mockito.ArgumentMatchers.anyString;
```

Comment:
Many of the changes were class name changes because Matcher is also used in Hamcrest.